cli/graph_util.rs

```diff
@@ -968,6 +1009,10 @@ impl ModuleGraphBuilder {
         check_js: CheckJsOption::Custom(self.tsconfig_resolver.as_ref()),
         exit_integrity_errors: true,
         allow_unknown_media_types,
+        ignore_graph_errors: matches!(
```

Comment:
We probably also need to do this for deno test and other subcommands that do typechecking

Reply:
Oh good point, yeah that makes sense to me!